docs/operations/shipperctl.rst

```diff
@@ -183,3 +183,95 @@ underscores in their names. To solve this, specify ``context`` explicitly in ``c
     context: gke_ACCOUNT_ZONE_CLUSTERNAME_APP_2 # and here
     scheduler:
       unschedulable: true
+
+
+Creating backups and restoring Using ``shipperctl backup`` Commands
+----------------------------------------------------------------------
+
+.. _create_backup:
+
+``shipperctl backup prepare``
++++++++++++++++++++++++++++++++
+
+1. The backup must be created by a `shipperctl` command. This guarantees you can restore this backup.
+Acquire a backup file by running
+
+.. code-block:: bash
+
+    $ kubectl config use-context mgmt-dev-cluster ##be sure to switch to correct context of the management cluster before backing up
+    Switched to context "mgmt-dev-cluster"
+    $ shipperctl backup prepare -v -f bkup-dev-29-10.yaml
+    NAMESPACE  RELEASE NAME              OWNING APPLICATION
+    default    super-server-dc5bfc5a-0   super-server
+    default2   super-server2-dc5bfc5a-0  super-server2
+    default3   super-server3-dc5bfc5a-0  super-server3
+    Backup objects stored in "bkup-dev-29-10.yaml"
+
+
+.. epigraph::
+
+    The commands default format is yaml. This will create a file named "bkup-dev-29-10.yaml" and store the backup there in a yaml format.
```

Comment:
```suggestion
    The commands' default format is yaml. This will create a file named "bkup-dev-29-10.yaml" and store the backup there in a yaml format.
```